packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -95,7 +95,8 @@ class CallContext {
   }
 
   public updateClientState(clientState: CallClientState): void {
-    const call = clientState.calls.get(this.callId ?? '');
+    const callId = this.callId ?? '';
+    const call = clientState.calls.find((candidate) => candidate.id === callId);
```

Comment:
nit: candidate -> call
```suggestion
    const call = clientState.calls.find((call) => call.id === callId);
```
(or callCandidate if call is being used too much, just something that indicates the variable is a call)

Reply:
I'm going to resist this request due to my love for shorter variable names and dependence on typing / IDEs (it's a tiny lambda function and any IDE worth its money (or free!) should show us the types).

`call` would be ideal, but it's frequently used as the captured variable we're comparing with.
To be honest, my inclination here would be to use `c`, coming from working in Go for a few years. But I'm playing fair :D

Reply:
I'd actually rather `c` than `candidate`, the issue I had with just candidate is that it seems like the variable has semantic meaning that this is a "candidate" of a call (making it sound like a participant/user)
(but also yeah its a tiny lambda function so more than happy to keep it as it is)

Reply:
Changed to `c`.